recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0a" %}
```

Comment:
Alpha versions should go in `conda-forge/label/libcugraph_dev`. See docs on how to configure conda-smithy to upload to a label.

Reply:
Thanks, I've added the following to `conda_build_config.yaml` in the next commit:
```
channel_targets:
  - conda-forge dev
```
Note: libcugraph will be out of alpha in a few days, so I'll likely be removing this and the "`a`" once the `v0.19.0` tag on the repo is available.